staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -904,7 +913,7 @@ func installAPI(s *GenericAPIServer, c *Config) {
 	routes.Version{Version: c.Version}.Install(s.Handler.GoRestfulContainer)
 
 	if c.EnableDiscovery {
-		s.Handler.GoRestfulContainer.Add(s.DiscoveryGroupManager.WebService())
+		s.Handler.GoRestfulContainer.Add(s.DiscoveryRootManager.WebService())
```

Comment:
With the featuregate off, I expected to see no change to this codepath.

Reply:
With the feature gate off, the DiscoveryRootManager will be proxying the request to the DiscoveryGroupManager which is effectively the same code path (but obviously has code changes). Would that be acceptable or do we need to strictly ensure that the code is not touched?